Dialog/CreditEditor_TableView.cpp

```diff
@@ -0,0 +1,237 @@
+
+                    QModelIndex newindex=this->model()->index(row,column);
+                    this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+
+                }
+            }
+        }
+    } else if (keycode == Qt::Key_Left) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            column--;
+
+            if (column >= 0) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    } else if (keycode == Qt::Key_Right || Qt::Key_Space) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            column++;
+
+            if (column < 32) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    } else if (keycode == Qt::Key_Up) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            row--;
+
+            if (row >= 0) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    } else if (keycode == Qt::Key_Down) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            row++;
+
+            if (row < 20) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    }
+}
+
+
+/// <summary>
+/// Delete function used (called by delete key or right click->delete)
+/// Delete is consistent across theme
+/// </summary>
+/// <param name="indexToDelete">
+/// An index of the cell to delete
+/// </param>
+void CreditEditor_TableView::deleteFunction(QModelIndex indexToDelete) {
+    int light = 0;
+    if(light != SettingsUtils::GetKey(static_cast<SettingsUtils::IniKeys>(6)).toInt()) {
+        this->model()->setData(indexToDelete, QColor(53, 54, 55), Qt::BackgroundRole); // perhaps inconsistant, idk, but looks not bad
```

Comment:
As the comment suggests, this is inconsistent. Let's use a `#define` color string to instantiate the QColor

Reply:
there are 2 close colors in the Theme file looks really similar, idk which one the background should use, so i just pick one of them. that's the meaning of the "inconsistent", the new fix looks good, just leave it there.

Reply:
What I meant by inconsistent was the instantiation of QColor (one of them is with int int int, one with with string)